code/modules/artifice/deadringer.dm

```diff
@@ -0,0 +1,183 @@
+/obj/item/weapon/deadringer
+	name = "silver pocket watch"
+	desc = "A fancy silver-plated digital pocket watch. Looks expensive."
+	icon = 'icons/obj/deadringer.dmi'
+	icon_state = "deadringer"
+	w_class = ITEMSIZE_SMALL
+	slot_flags = SLOT_ID | SLOT_BELT | SLOT_TIE
+	origin_tech = list(TECH_ILLEGAL = 3)
+	var/activated = 0
+	var/timer = 0
+	var/bruteloss_prev = 999999
+	var/fireloss_prev = 999999
+	var/mob/living/carbon/human/corpse = null
+	var/mob/living/carbon/human/watchowner = null
+
+
+/obj/item/weapon/deadringer/New()
+	..()
+	processing_objects |= src
+
+/obj/item/weapon/deadringer/Destroy() //just in case some smartass tries to stay invisible by destroying the watch
```

Comment:
You need to remove the item from the processing_objects list somewhere in this proc.

Reply:
done